src/test/kotlin/sirius/biz/cluster/RedisUserMessageCacheTest.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.cluster
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import sirius.kernel.SiriusExtension
+import sirius.kernel.di.std.Part
+import sirius.web.controller.Message
+import sirius.web.controller.MessageLevel
+import sirius.web.http.DistributedUserMessageCache
+import java.util.*
```

Comment:
Wollen wir den Wildcard in Kotlin?

Reply:
I think we have to set up the Kotlin style/formatting rules correctly in sirius-parent.
There are official coding conventions that are available and should be applied.

As other imports are not converted to wildcard imports my guess is that only `java.util` is allows to be imported as wildcard.

Reply:
Would also suggest to enable the "wildcard imports" inspection in sirius parent :)

Reply:
Here is a link to the style guide: https://kotlinlang.org/docs/coding-conventions.html

Reply:
The main problem that arises with star imports in Java (name clashes) is actually not present in kotlin because there is this small feature:
`import org.test.Message as TestMessage`